srcs/exec/exec_create_process_args.c

```diff
@@ -0,0 +1,75 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   exec_create_process_args.c                         :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: jbrinksm <jbrinksm@student.codam.nl>         +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2019/11/04 18:19:28 by jbrinksm       #+#    #+#                */
+/*   Updated: 2019/11/05 10:55:18 by jbrinksm      ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "vsh.h"
+
+static size_t	count_args(t_ast *ast)
+{
+	t_ast	*probe;
+	size_t	i;
+
+	i = 0;
+	if (ast == NULL || ast->type != WORD)
```

Comment:
We might need to fix this in case the first ast node is an assign that is actually an executable. Will do in other branch.